Bottom: 3456a5fb8b1d34653e919bfb859090a9479837c3
Top:    1b2225773482b4cab02ab7cd118a55a577fd9aa0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-03 23:37:55 +0000

Refresh of update-the-crosshair-constrain

---

diff --git a/src/buffer.c b/src/buffer.c
index 425b52b..d311420 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -1226,7 +1226,7 @@ RotateBuffer (BufferType *Buffer, BYTE Number)
   /* finally the origin and the bounding box */
   ROTATE (Buffer->X, Buffer->Y, Buffer->X, Buffer->Y, Number);
   RotateBoxLowLevel (&Buffer->BoundingBox, Buffer->X, Buffer->Y, Number);
-  SetCrosshairRangeToBuffer (Buffer);
+  SetCrosshairRangeToBuffer ();
 }
 
 static void
@@ -1361,7 +1361,7 @@ FreeRotateBuffer (BufferType *Buffer, Angle angle)
   ENDALL_LOOP;
 
   SetBufferBoundingBox (Buffer);
-  SetCrosshairRangeToBuffer (Buffer);
+  SetCrosshairRangeToBuffer ();
 }
 
 
@@ -1477,7 +1477,7 @@ MirrorBuffer (BufferType *Buffer)
   }
   ENDALL_LOOP;
   SetBufferBoundingBox (Buffer);
-  SetCrosshairRangeToBuffer (Buffer);
+  SetCrosshairRangeToBuffer ();
 }
 
 
@@ -1604,6 +1604,7 @@ SwapBuffer (BufferType *Buffer)
 	}
     }
   SetBufferBoundingBox (Buffer);
+  SetCrosshairRangeToBuffer ();
 }
 
 /* ----------------------------------------------------------------------
